backend/src/openbeheer/zaaktype/api/views.py

```diff
@@ -224,6 +224,10 @@ class ZaakTypeDetailView(DetailView[ExpandableZaakType]):
     endpoint_path = "zaaktypen/{uuid}"
     has_versions = True
 
+    @staticmethod
+    def _get_params_with_status(zaaktype: ZaakType):
+        return {"zaaktype": zaaktype.url, "status": "alles"}
+
```

Comment:
I added this because otherwise any related resource that is still a concept was not being expanded